### ISSUE TITLE
Language pack inheritance order

### ISSUE THREAD

**Issue by koenner01** [ho-nl/magento2-nl_NL#7]
Is there a reason why in language.xml the Magento vendor inheritance is commented ?
Because currently if we are adding the ho/nl_NL pack as a parent to our projects pack, the order in packs is:

Ho|nl_NL
Magento|nl_NL
Project|nl_NL

Aren't the translations in this pack supposed to replace the 'faulty' Magento ones ?

**Comment by JeroenVanLeusden** [comment]
As far as I understand: Magento doesn't offer translations it's self, so it doesn't matter if we add the inheritance. If you believe this should be different, could you please explain?

**Comment by koenner01** [comment]
It matters because the inheritance in packs will set incorrect translations on the frontend if the module does not depend on Magento.

Magento does provide a language pack for nl_nl although it's empty (for now).
https://github.com/magento/magento2/tree/develop/app/i18n/Magento/nl_NL

What we do is:

- Add the ho_nl pack
- Create a module that contains multiple nl_nl language packs with (conditional) additions and changed translations; This pack has parent pack ho-nl.
- Add a language pack for the project (in app/i18n/<projectname>/nl_nl) for which the parent pack is 1 or more packs from our module.

The expected order for translations would be:

- Magento nl_NL
- Ho nl_NL
- <customModule> nl_NL
- project nl_NL

But this is not the case.

**Comment by koenner01** [comment]
@JeroenVanLeusden there is a typo in the language.xml
The vendor should be 'Magento' (with capital M) and the package should be 'nl_NL' (with capital NL in the second part).

You can check the inheritance in \Magento\Framework\App\Language\Dictionary in the function getDictionary(). Just dump the $packs variable after this code

```
// Collect the inherited packages with meta-information of sorting
$packs = [];
foreach ($languages as $languageConfig) {
    $this->collectInheritedPacks($languageConfig, $packs);
}
uasort($packs, [$this, 'sortInherited']);
```

**Comment by JeroenVanLeusden** [comment]
Thank you for the clarification, this has been fixed on the develop branch(ae8aa48510c0726f0835e6e8d04ff6eec258673e) and will be included in the next release.
